docs/DeveloperGuide.md

```diff
@@ -289,42 +289,42 @@ _{Explain here how the data archiving feature will be implemented}_
+| `*`      | user short on time                         | visit households that are geographically close in the same day/visit       | minimise the travelling time for myself                                                                |
+| `*`      | user switching devices                     | transfer the SocialBook contacts from one device to another                | avoid having to manually re-enter all the current contacts                                             |
+| `*`      | max/expert user                            | export all contacts to a txt file/excel file                               | so that I have a copy of all beneficiaries, past and present, in a larger database                     |
+| `*`      | max/expert user                            | record social media credentials of a contact, if they have any             | so that I can get to know their lives better and establish a closer bond                               |
+| `*`      | max/expert user                            | send reminder to my email a day before my visit                            | so that I will not forget to visit a household                                                         |
+| `*`      | max/expert user                            | upload pictures of my beneficiaries                                        | so that I will not forget how they look like                                                           |
+| `*`      | max/expert user                            | view contacts on google maps                                               | so that I can directly see where and how to go to the household                                        |
+| `*`      | visual rememberer                          | add photos for each contact                                                | to match beneficiary names to their faces more easily                                                  |
+| `*`      | loyal user                                 | save contact information into a shareable form                             | share contacts with colleagues as need be                                                              |
+| `*`      | experienced social worker                  | locate a link where I can share my views and suggestions on the app        | help the app be more user centered and helpful over time                                               |
+| `*`      | efficient person                           | create visit paths based on proximity of beneficiaries                     | to be able to serve the most beneficiaries a day                                                       |
+| `*`      | social person                              | add notes of on conversations with beneficiaries                           | develop stronger relationships by building rapport                                                     |
```

Comment:
what do you think about the suggestion i made under issue #305

Reply:
Tbh I think we should just stick to this. Not much of a difference and I believe the "so that I can is what was taught in class".